app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,23 @@
+.row.comment {
+  border: 1px solid grey;
+  margin: 15px;
+  padding: 15px;
+
+  h5{
+    font-size: 25px;
+    font-weight: bold;
+  }
+
+  .content{
+    background-color: $white;
```

Comment:
you are using variables in wrong way.
do you think variable "$white" will have value "red"? or other color? white is white.
if you want to use variables - use better names, like "$content_bg_color"